### DISCUSSION TITLE
azure-blob: Is there any particular reason why scaler doesn't check sub-folders?

### DISCUSSION THREAD

**Discussion by jbaranov** [kedacore/keda#1300] [General]
We decided to try azure-blob scaler recently and realized that it doesn't traverse subfolders. After digging into the code I realized that the code was written in a way that forces the`blobPrefix` value to `/` if empty or no value was configured

![image](https://user-images.githubusercontent.com/999394/97695920-76d0ec80-1aad-11eb-9867-b3be1bfe835f.png)

![image](https://user-images.githubusercontent.com/999394/97695870-6882d080-1aad-11eb-8eea-97413edc7064.png)

Is there any particular reason to prevent the scaler from checking subfolders within container?

**Comment by zroubalik** [comment]
I am not Azure expert, so I don't see any particular reason behind this. It just might be not implemented.

I think that there's already issue opened for this: https://github.com/kedacore/keda/issues/1210

Feel free to comment on that issue.

**Reply by zroubalik** [reply]
@anirudhgarg @ahmelsayed @jeffhollan ^
